clients/egui/src/account/mod.rs

```diff
@@ -29,38 +32,69 @@ pub struct AccountScreen {
     update_tx: mpsc::Sender<AccountUpdate>,
     update_rx: mpsc::Receiver<AccountUpdate>,
 
+    save_req_tx: mpsc::Sender<SaveRequest>,
+
     tree: FileTree,
     sync: SyncPanel,
     usage: Result<Usage, String>,
     workspace: Workspace,
     modals: Modals,
+    shutdown: Option<AccountShutdownProgress>,
```

Comment:
potentially could just be a bool?

Reply:
There'll be other things we add that we'd want to track on shutdown, for example, if there's a sync currently happening, finish up, etc.